main/guides/integration/name-services.md

```diff
@@ -0,0 +1,106 @@
+# Name Services
+
+The Agoric chain has a few name services:
+ - `agoricNames` for well-known names controlled by governance
+ - `namesByAddress` - a namespace for each provisioned account
+ - `board` - to map objects to arbitrary names and back
+
+## agoricNames / agoricNamesAdmin - well known names
+
+To look up well-known objects, such as the BLD brand:
+
+```js
+const brandHub = E(agoricNames).lookup('brand');
+const bldBrand = await E(brandHub).lookup('BLD');
+```
+
+or, equivalently:
+
+```js
+const bldBrand = await E(agoricNames).lookup('brand', 'BLD');
+```
+
+`agoricNames` is a _NameHub_. In general, `E(hub).lookup(key1, key2, ...rest)` is equivalent
+to `E(E(hub).lookup(key1)).lookup(key2, ...rest)`. The NameHub interface also includes `Map`-like
+methods `has`, `entries`, `keys`, and `values`.
+
+The keys of `agoricNames` include:
+ - `brand` - IST, BLD, Invitation (Zoe invitation brand), timer (a TimerBrand, not an ERTP brand)
+ - `issuer` - IST, BLD, Invitation, ...
+ - `vbankAsset` - IST, BLD, USDC, ATOM, ... - including info about corresponding cosmos denom
```

Comment:
TODO: give the type of the values of the `vbankAsset` namehub

Reply:
and/or link to https://docs.agoric.com/guides/getting-started/contract-rpc.html#vbank-assets-and-cosmos-bank-balances